app/assets/stylesheets/administrate/components/_header.scss

```diff
@@ -6,7 +6,7 @@
 }
 
 .header__heading {
-  font-size: modular-scale(3);
+  font-size: 1.728em; // 1.2 ** 3
```

Comment:
We can simplify this to `1.75em`; it doesn’t have to match the output of the old `modular-scale` value exactly. We can also remove the comment.